pep/app/pods/most-cited/route.ts

```diff
@@ -13,14 +16,15 @@ export default class MostCited extends PageNav(Route) {
      * @returns {Promise<Document[]>}
      * @memberof MostCited
      */
-    async model() {
-        const queryParams = buildQueryParams({
+    async model(queryParams: SearchParams) {
```

Comment:
you should probably create a new interface for this page's specific params instead of using the search/route's params interface, as I don't think they actually match up.